### ISSUE TITLE
Page for editing user settings 1

### ISSUE THREAD

**Issue by breuert** [stella-project/stella-server#15]
- [x] @jueri Add interface for editing username, email, password
- [x] @jueri Implement code for writing changes into database
- [x] @jueri Remove unneccessary headings and correct misspellings
- [x]  @narges1212, testing the page

**Comment by narges1212** [comment]
@breuert I noticed that Stella-server-master_web is not up at all. I tried to make it up again but didn't work. Maybe there is something wrong with my server.
so sorry I couldn't test it.

stella-server_nginx      "/docker-entrypoint.…"   2 minutes ago       Up 2 minutes    0.0.0.0:80->80/tcp       stella-server_nginx_1
stella-server_web  "bash -c 'python set…"   2 minutes ago       **Restarting (1) 33 seconds ago**         stella-server_web_1
postgres        "docker-entrypoint.s…"   2 minutes ago       Up 2 minutes                    0.0.0.0:5432->5432/tcp   stella-server_db_1
